wenet/bin/recognize.py

```diff
@@ -128,10 +128,10 @@ def main():
     symbol_table = read_symbol_table(args.dict)
     test_conf = copy.deepcopy(configs['dataset_conf'])
 
-    test_conf['filter_conf']['max_length'] = 102400
-    test_conf['filter_conf']['min_length'] = 0
-    test_conf['filter_conf']['token_max_length'] = 102400
-    test_conf['filter_conf']['token_min_length'] = 0
+    test_conf['filter_conf']['max_length'] = 1200
```

Comment:
why change the default parameter here? is there any limits on IPU?

Reply:
is reset to default, a wrong change in the very beginning